tests/dragonfly/instance.py

```diff
@@ -416,7 +416,7 @@ def create(self, existing_port=None, path=None, version=100, **kwargs) -> DflyIn
         args.setdefault("noversion_check", None)
         # MacOs does not set it automatically, so we need to set it manually
         args.setdefault("maxmemory", "8G")
-        vmod = "dragonfly_connection=1,accept_server=1,listener_interface=1,main_service=1,rdb_save=1,replica=1,cluster_family=1,proactor_pool=1,dflycmd=1"
+        vmod = "dragonfly_connection=1,accept_server=1,listener_interface=1,main_service=1,rdb_save=2,replica=1,cluster_family=1,proactor_pool=1,dflycmd=1,snapshot=2"
```

Comment:
are you sure you want to keep it? it will produce rather big logs.

Reply:
I will revert this